packages/contracts-bedrock/src/dispute/FaultDisputeGame2.sol

```diff
@@ -833,7 +839,7 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         returns (ClaimData storage ancestor_)
     {
         // Grab the trace ancestor's expected position.
-        Position traceAncestorPos = _global ? _pos.traceAncestor() : _pos.traceAncestorBounded(SPLIT_DEPTH);
+        Position traceAncestorPos = _global ? _pos.traceAncestorBounded(nBits - 1) : _pos.traceAncestorBounded(SPLIT_DEPTH);
```

Comment:
I am not sure `_pos.traceAncestorBounded(nBits - 1)` is correct for both `nBits == 1` and `nBits > 1` cases.

For example, `traceAncestor(3) = 1`, while `traceAncestorBounded(3, 0) = 3`